### PR TITLE
Insufficient interface error handling

### PR DESCRIPTION
**Pull request by AhmedKorim**

### CODE REVIEW

File: packages/react-environment/src/WebbProvider.tsx [hunk 1]
```diff
@@ -294,13 +305,14 @@ export const WebbProvider: FC<WebbProviderProps> = ({ applicationName = 'Webb Da
...
         defaultAccount = defaultAccount ?? accounts[0]?.address;
         const defaultFromSettings = accounts.find((account) => account.address === defaultAccount);
+        console.log(defaultFromSettings, 'defaultFromSettings');
         if (defaultFromSettings) {
           _setActiveAccount(defaultFromSettings);
           await activeApi.accounts.setActiveAccount(defaultFromSettings);
           networkStorage?.set('networksConfig', {
             ...networkDefaultConfig,
             [chainConfig.id]: {
-              ...chainConfig[chainConfig.id],
```
Comment (nepoche) [inline packages/react-environment/src/WebbProvider.tsx:303]: Why this change here?

Comment (AhmedKorim) [reply packages/react-environment/src/WebbProvider.tsx:303]: This if for change to be just `...chianConfig` , on init restoring the default account or the first one on the accounts list to the storage
